Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -496,6 +497,10 @@ void ABenchmarkGymGameMode::GenerateTestScenarioLocations()
 			FVector PointA = PlayerStream.VRand()*RoamRadius;
 			FVector PointB = PlayerStream.VRand()*RoamRadius;
 			PointA.Z = PointB.Z = 0.0f;
+			if (FVector::DistSquared2D(PointA, PointB) < MinDistanceSq)
+			{
+				PointA *= -1.f;
```

Comment:
Does this have the possibility of causing characters to move out of the "density regions"?
Perhaps we just generate random points till we find a valid pair?

Reply:
PointA and B are relative vectors of the same length (RoamRadius) that are added onto an origin position to generate the two waypoint coordinates. If we generated infinite points and plotted them, we'd get a perfect circle. This is just checking that if the two points that we've generated onto that circle, are too close to one another, flip the position of the first point (so that it's on the other side of the circle).

Reply:
Worth a comment?